scripts/generate_topographic_data.py

```diff
@@ -0,0 +1,87 @@
+"""
+Script that takes the SRTM 30m worldwide elevation maps and creates a
+single representation of the area in NetCDF and GeoTIFF formats
+
+The SRTM files can be downloaded from NASA, and are in CRS EPSG:4326
+"""
+import os.path
+import glob
+import rasterio
+from rasterio.merge import merge
+from rasterio.warp import Resampling
+from nowcasting_dataset.geospatial import OSGB
+import rioxarray
+
+dst_crs = OSGB
+
+# Go through, open all the files, combined by coords, then save out to NetCDF, or GeoTIFF
+files = glob.glob("/run/media/jacob/data/SRTM/*.tif")
+out_dir = "/run/media/jacob/data/SRTM1KM/"
```

Comment:
this might have to be more general, shouldnt have your name in it

Reply:
Yeah, I need to move the raw files to GCS and AWS, and I'll update it

Reply:
Currently should finish with the raw data in about 5 hours

Reply:
Ooh... please could you add a comment in the script to say how long it usually takes to run?